promdump/promdump.go

```diff
@@ -43,6 +47,20 @@ func writeFile(values *[]*model.SampleStream, fileNum uint) error {
 	return ioutil.WriteFile(filename, valuesJSON, 0644)
 }
 
+func cleanFiles() error {
+	files, err := filepath.Glob(fmt.Sprintf("%s.[0-9][0-9][0-9][0-9][0-9]", *out))
+	if err != nil {
+		return err
+	}
+	for _, file := range files {
+		err = os.Remove(file)
+		if err != nil {
```

Comment:
Perhaps we can print something about the removes.  Per our conversation, maybe not ALL removes, but a summary of how many, etc.